internal/cmd/binding/get_binding_test.go

```diff
@@ -55,50 +54,119 @@ var _ = Describe("Get binding command test", func() {
 		return commandToRun.Execute()
 	}
 
-	Context("when no binding name is provided", func() {
-		It("should return error", func() {
-			client.GetBindingByIDReturns(&binding, nil)
-			err := executeWithArgs("")
+	Describe("Get service binding", func() {
+		BeforeEach(func() {
+			client.GetInstanceByIDReturnsOnCall(0, &instance1, nil)
+			client.GetInstanceByIDReturnsOnCall(1, &instance2, nil)
+		})
+		Context("when no binding name is provided", func() {
+			It("should return error", func() {
+				client.GetBindingByIDReturns(&binding, nil)
+				err := executeWithArgs("")
 
-			Expect(err).Should(HaveOccurred())
+				Expect(err).Should(HaveOccurred())
+			})
 		})
-	})
 
-	Context("when more than one binding with same name exists", func() {
-		var response *types.ServiceBindings
-		BeforeEach(func() {
-			response = &types.ServiceBindings{ServiceBindings: []types.ServiceBinding{binding, binding2}, Vertical: true}
-			client.ListBindingsReturns(response, nil)
+		Context("when more than one binding with same name exists", func() {
+			var response *types.ServiceBindings
+			BeforeEach(func() {
+				response = &types.ServiceBindings{ServiceBindings: []types.ServiceBinding{binding, binding2}, Vertical: true}
+				client.ListBindingsReturns(response, nil)
+			})
+
+			It("should return both bindings", func() {
+				client.GetBindingByIDReturnsOnCall(0, &binding, nil)
+				client.GetBindingByIDReturnsOnCall(1, &binding2, nil)
+				err := executeWithArgs("binding1")
+				Expect(err).ShouldNot(HaveOccurred())
+
+				Expect(buffer.String()).To(ContainSubstring(response.TableData().String()))
+			})
 		})
 
-		It("should return both bindings", func() {
```

Comment:
should this be removed?

Reply:
it's not removed, it under section : "Describe("Get service binding", func() {"